docs/content/connectors/oceanbase-cdc(ZH).md

```diff
@@ -0,0 +1,238 @@
+    'database-name' = 'test_db',
+    'table-name' = 'orders',
+    'hostname' = '127.0.0.1',
+    'port' = '2881',
+    'rootserver-list' = '127.0.0.1:2882:2881',
+    'logproxy.host' = '127.0.0.1',
+    'logproxy.port' = '2983');
+
+-- 从表 orders 中读取快照数据和 binlogs
+Flink SQL> SELECT * FROM orders;
+```
+
+您也可以访问 Flink 官网文档，快速体验将数据从 OceanBase 导入到 Elasticsearch。更多信息，参考 [Flink 官网文档](https://ververica.github.io/flink-cdc-connectors/release-2.2/content/%E5%BF%AB%E9%80%9F%E4%B8%8A%E6%89%8B/oceanbase-tutorial-zh.html)。
+
+## Flink OceanBase CDC 连接器选项
+
+选项 | 是否必选 | 默认值 | 类型 | 描述
+---- | ----- | ------ | ----- | ----
+connector | 是 | 无 | String | 指定要使用的连接器。此处为 `mysql-cdc`。
+scan.startup.mode | 是 | 无 | String | 指定 Flink OceanBase CDC 消费者的启动模式。可取值为 `initial`、`latest-offset` 或`timestamp`。
+scan.startup.timestamp | 否 | 无 | Long | 起始点的时间戳，单位为秒。仅在 `scan.startup.mode` 的值为 `timestamp` 时适用。
+username | 是 | 无 | String | 连接 OceanBase 数据库的用户的名称。
+password | 是 | 无 | String | 连接 OceanBase 数据库时使用的密码。
+tenant-name | 是 | 无 | String | 待监控 OceanBase 数据库的租户名。
+database-name | 是 | 无 | String | 待监控 OceanBase 数据库的数据库名。
+table-name | 是 | 无 | String | 待监控 OceanBase 数据库的表名。
+hostname | 否 | 无 | String | OceanBase 数据库或 OceanBbase 代理 ODP 的 IP 地址或主机名。
+port | 否 | 无 | String | Integer | OceanBase 数据库服务器的整数端口号。可以是 OceanBase 服务器的 SQL 端口号，默认值为 `2881`。或 ODP 的端口号默认值为 `2883`。
+connect.timeout |  否 | 30s | Duration | 连接器在尝试连接到 OceanBase 数据库服务器后的超时时间。
+server-time-zone |  否 | UTC | String | 数据库服务器中的会话时区，例如 `"Asia/Shanghai"`。此选项控制 OceanBase 数据库中的 `TIMESTAMP` 类型在快照读取时如何转换为`STRING`。确保此选项与 `oblogproxy` 的时区设置相同。
+rootserver-list | 是 | 无 | String | 格式为 `ip:rpc_port:sql_port`。多个服务器地址使用英文分号 `;` 隔开。
+logproxy.host | 是 | 无 | String | oblogproxy 的 IP 地址或主机名。
+logproxy.port | 是 | 无 | Integer | oblogproxy 的端口号。
+
+## 支持的元数据
+
+在创建表时，您可以使用以下格式的元数据作为只读列（VIRTUAL）。
+
+Key | 数据类型 | 描述
+----- | ----- | -----
+tenant_name | STRING NOT NULL | 当前记录所属的租户名称。
+database_name | STRING NOT NULL | 当前记录所属的数据库名称。
+table_name | STRING NOT NULL | 当前记录所属的表名称。
+op_ts | TIMESTAMP_LTZ(3) NOT NULL | 当前记录表在数据库中更新的时间。
+如果从表的快照而不是 binlog 读取记录，该值将始终为 0。
+
+例如：
+
+```sql
+CREATE TABLE products (
+    tenant_name STRING METADATA FROM 'tenant_name' VIRTUAL,
+    db_name STRING METADATA FROM 'database_name' VIRTUAL,
+    table_name STRING METADATA  FROM 'table_name' VIRTUAL,
+    operation_ts TIMESTAMP_LTZ(3) METADATA FROM 'op_ts' VIRTUAL,
+    order_id INT,
+    order_date TIMESTAMP(0),
+    customer_name STRING,
+    price DECIMAL(10, 5),
+    product_id INT,
+    order_status BOOLEAN,
+    PRIMARY KEY(order_id) NOT ENFORCED
+) WITH (
+   'connector' = 'oceanbase-cdc',
+   'scan.startup.mode' = 'initial',
+   'username' = 'user@test_tenant',
+   'password' = 'pswd',
+   'tenant-name' = 'test_tenant',
+   'database-name' = 'test_db',
+   'table-name' = 'orders',
+   'hostname' = '127.0.0.1',
+   'port' = '2881',
+   'rootserver-list' = '127.0.0.1:2882:2881',
+   'logproxy.host' = '127.0.0.1',
+   'logproxy.port' = '2983');
```

Comment:
The English version have the option 'working-mode' = 'memory'.

Reply:
Your source?
I'm using https://github.com/ververica/flink-cdc-connectors/blob/release-2.2/docs/content/connectors/oceanbase-cdc.md